manifest/manifest.go

```diff
@@ -27,6 +27,7 @@ var (
 		// MaxBackoff is 15min given default params
 		BaseDecisionBackoffTable: []float64{1.3, 1.69, 2.2, 2.86, 3.71, 4.83, 6.27, 7.5},
 		HeadLookback:             0,
+		Finalize:                 true,
```

Comment:
Would be nice to make the default be zero-value of `bool`

Reply:
I generally agree but... I think this way is actually safer in this case. But I could be swayed either way.